src/main/java/com/iprd/fhir/utils/Utils.java

```diff
@@ -574,4 +574,15 @@ public static boolean noneMatchDates(List<Date> dates, Date currentDate) {
 			.noneMatch(date -> date.equals(currentLocalDate));
 	}
 
+	public static Double anonymizedData(double actualValue, double minPercent, double maxPercent) {
+		double noisePercent = getRandomPercent(minPercent, maxPercent);
+		double noiseValue = (actualValue * noisePercent / 100);
+		double anonymizedValue = actualValue + noiseValue;
```

Comment:
i see noise being only added, we had a discussion of subtracting.

Reply:
I thought that was just a suggestion. Yet I have modified the function to do the same now. Thanks.